CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contribution guidelines
+
+First off, thank you for considering contributing to delta.
+
+If your contribution is not straightforward, please first discuss the change you
+wish to make by creating a new issue before making the change.
+
+## Developing
+
+### Set up
+
+This is no different than other Rust projects.
+
+```shell
+git clone https://github.com/dandavison/delta/
+cd delta
+cargo build
+```
+
+### Useful Commands
+
+- Build and run release version:
+
+  ```shell
+  cargo build --release && cargo run --release
+  ```
```

Comment:
I'm not sure `cargo run --release` is helpful seeing as it requires stdin or two input file args.